common/crypto/crypto_test.go

```diff
@@ -56,37 +33,18 @@ func TestGetMD5(t *testing.T) {
 
 func TestGetSHA512(t *testing.T) {
 	t.Parallel()
-	originalString := []byte("I am testing the GetSHA512 function in common!")
-	expectedOutput := []byte(
-		`a2273f492ea73fddc4f25c267b34b3b74998bd8a6301149e1e1c835678e3c0b90859fce22e4e7af33bde1711cbb924809aedf5d759d648d61774b7185c5dc02b`,
-	)
-	actualOutput, err := GetSHA512(originalString)
-	if err != nil {
-		t.Fatal(err)
-	}
-	actualStr := HexEncodeToString(actualOutput)
-	if !bytes.Equal(expectedOutput, []byte(actualStr)) {
-		t.Errorf("Expected '%x'. Actual '%x'",
-			expectedOutput, []byte(actualStr))
-	}
+	h, err := GetSHA512([]byte("I am testing the GetSHA512 function in common!"))
+	require.NoError(t, err, "GetSHA512 must not error")
+	const expectedOutput = "a2273f492ea73fddc4f25c267b34b3b74998bd8a6301149e1e1c835678e3c0b90859fce22e4e7af33bde1711cbb924809aedf5d759d648d61774b7185c5dc02b"
+	assert.Equal(t, expectedOutput, hex.EncodeToString(h), "GetSHA512 result should match the expected output")
```

Comment:
🚧 Couldn't we just use `sha512.Sum512()` function and not have `GetSHA512` or this test at all?
Same with `md5.Sum()` and sha256.Sum256()`